sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuParquetFileFormat.scala

```diff
@@ -122,27 +121,24 @@ object GpuParquetFileFormat {
     }
 
     SparkShimImpl.int96ParquetRebaseWrite(sqlConf) match {
-      case "EXCEPTION" =>
-      case "CORRECTED" =>
-      case "LEGACY" =>
+      case DateTimeRebaseException.value | DateTimeRebaseCorrected.value => // Good
```

Comment:
Why not have the match switch on the result of rebaseModeFromName rather than reach into the value on each object?  Comment applies to all the similar '.value' accesses below and in other files.